.github/workflows/build.yml

```
@@ -41,7 +41,7 @@ jobs:
           python -m pip install --upgrade pip wheel
           pip install -e .
           python -c "import autogen"
-          pip install pytest mock
+          pip install pytest mock networkx matplotlib
```

```suggestion
          pip install pytest mock
```

@kevin666aa, I have removed the visualization now that we don't need it, therefore don't need networkx nor matplotlib. :)